ci-operator/step-registry/upi/libvirt/cleanup/post/upi-libvirt-cleanup-post-commands.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# install jq if not installed
+if ! [ -x "$(command -v jq)" ]; then
```

Comment:
this is an outdated cleanup script. We should copy over the one that's currently used for IPI, with the added sleeps for safety.

Reply:
Also the pool stuff is unique to this version, so that needs to stay